tests/lib/rules/array-callback-return.js

```diff
@@ -239,6 +239,85 @@ ruleTester.run("array-callback-return", rule, {
         { code: "foo.forEach((x) => { if (a === b) { return x; } })", options: checkForEachAllowVoid, parserOptions: { ecmaVersion: 6 }, errors: [{ messageId: "expectedNoReturnValue" }] },
         { code: "foo.forEach((x) => { if (a === b) { return !x } })", options: checkForEachAllowVoid, parserOptions: { ecmaVersion: 6 }, errors: [{ messageId: "expectedNoReturnValue" }] },
 
+        {
+            code: "foo.forEach(x => x)",
+            options: checkForEachOptions,
+            parserOptions: { ecmaVersion: 6 },
+            errors: [{
+                messageId: "expectedNoReturnValue",
+                suggestions: [
+                    { output: "foo.forEach(x => {x})" }
```

Comment:
Can you add `messageId` to all suggestion test objects? Also, can you add `suggestions` to all test cases where the rule provides suggestions? This is for better testing and will soon be mandatory per [RFC #84](https://github.com/eslint/rfcs/blob/main/designs/2021-stricter-rule-test-validation/README.md).